homeassistant/components/bmw_connected_drive/strings.json

```diff
@@ -97,7 +97,21 @@
         "name": "Charging end time"
       },
       "charging_status": {
-        "name": "Charging status"
+        "name": "Charging status",
+        "state": {
+          "default": "Default",
+          "charging": "Charging",
+          "error": "Error",
+          "complete": "Complete",
+          "fully_charged": "Fully charged",
+          "finished_fully_charged": "Fully charged",
```

Comment:
Having 2 the same states doesn't make a lot of sense imo

Reply:
Yeah - the issue is we don't really know which of these enums could occur, they are collected over multiple years.
Therefore I decided in this case to combine both, but I can also have different translations if you feel this is needed.

Reply:
The thing I am worried about is that with this change when you edit an automation and select a possible state, you see "Fully charged" twice and never know which one is triggered when. So imagine created an automation for Fully charged 1, and then the car actually displays 2, that's strange.

Reply:
+1, I agree on this.

I'll translate it for now and then have to think later how to deal with this ever growing list.